src/site/antora/modules/ROOT/partials/manual/api-best-practice-dont-use-string-concat.adoc

```diff
@@ -34,3 +29,10 @@ Imagine `userId` being provided by the user with the following content:
 ----
 /* GOOD */ LOGGER.info("failed for user ID `{}`", userId);
 ----
+
+* [x] Use message lambdas
++
+[source,java]
+----
+/* GOOD */ LOGGER.info(() -> "failed for user ID: " + userId);
```

Comment:
@ppkarwasz, I think this is a foot gun. It'd be very hard to distinguish for inexperienced users, which is the chief audience of this guide, why it is fine to do string concatenation here, whereas before it was not. IMHO, this should not be supported by the API in the first place, but anyway.